src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/takemanagement/view/RecordScriptureFragment.kt

```diff
@@ -97,50 +44,22 @@ class RecordScriptureFragment : Fragment() {
                 }
                 //selected take and drag target
                 stackpane {
-                    addClass(RecordScriptureStyles.selectedTakeContainer)
                     // drag target glow
-                    stackpane {
-                        addClass(RecordScriptureStyles.dragTarget, RecordScriptureStyles.glow)
-                        visibleProperty().bind(draggingTakeProperty.booleanBinding { it != null })
-                    }
-                    vbox {
-                        alignment = Pos.CENTER
-
-                        // Check if the selected take card has changed
-                        isFillWidth = false
-                        val placeholder = vbox {
-                            addClass(RecordScriptureStyles.placeholder)
+                    add(dragComponents
```

Comment:
formatting

Reply:
These adds are gone now that I created a DragTarget control